pages/about.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EmpowerHer | About</title>
+    <link rel="stylesheet" href="../css/about.css">
+    <link rel="stylesheet" href="https://use.typekit.net/jgd5och.css">
+    <script src="https://kit.fontawesome.com/4dbfbae31e.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header>
+        <div>
+            <i class="fa-sharp fa-solid fa-bars fa-2xl menu-icon" style="color: #545454; padding: 2rem 1.5rem;"></i>
+        </div>
+        <section class="header-one">
+            <div class="header-one-container">
+                <ul>
+                    <li><i class="fa-brands fa-facebook-f" style="color: #ffffff;"></i></li>
+                    <li><i class="fa-brands fa-twitter" style="color: #ffffff;"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </section>
+        <section class="header-two">
+            <div class="header-two-container">
+                <a href="../index.html"><img src="../assets/images/Color logo - no background.png"
+                        alt="EmpowerHer icon"></a>
+                <nav>
+                    <ul>
+                        <li><a href="./pages/about.html" class="tag">About</a></li>
+                        <li><a href="./pages/about.html">Program</a></li>
+                        <li><a href="./pages/about.html">Join</a></li>
+                        <li><a href="./pages/about.html">Sponsor</a></li>
+                        <li><a href="./pages/about.html">News</a></li>
+                        <li><a href="./pages/about.html" class="sponsor tag">Sponsor</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </section>
+        <section class="mobile-menu">
+            <i class="fa fa-close" style="color:#ffffff;"></i>
+            <ul>
+                <li><a href="../index.html" class="tag">Home</a></li>
+                <li><a href="../index.html">Program</a></li>
+                <li><a href="../index.html">Join</a></li>
+                <li><a href="../index.html">Sponsor</a></li>
+                <li><a href="../index.html">News</a></li>
+                <li><a href="../index.html" class="sponsor tag">Sponsor</a></li>
+            </ul>
+        </section>
+    </header>
+
+    <main>
+        <section class="hero-section">
+            <section class="hero-article">
+                <h1>EmpowerHer Summit 2023:</h1>
+                <h2>Building the level ground for growth</h2>
+                <hr class="pinkhr">
```

Comment:
- [ ] Kindly make sure you stick to the original design as much as possible. your typo(fonts size, family ...) are supposed to match the original design.
### Original design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/b2ae39f4-414a-46d7-a35c-8846330b90fa)
### Your design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/0f20e6d4-d150-444c-b1c6-67298b36e129)